scripts/fastaq_filter.py

```diff
@@ -0,0 +1,55 @@
+from typing import List, Union
+
+
+def gc_test(seq: str, gc_bounds: tuple) -> bool:
+    """
+    Check sequence for GC-content.
+    :param seq: given DNA sequence for analysis (str)
+    :param gc_bounds: given threshold for GC-content (tuple)
+    :return: True if sequence is within given threshold (bool)
+    """
+    gc_amount = (seq.count('G') + seq.count('C')) / len(seq) * 100
+    if gc_amount < gc_bounds[0] or gc_amount > gc_bounds[1]:
```

Comment:
Cравнение можно записать таким образом:
```suggestion
    if gc_bounds[1] < gc_amount < gc_bounds[0] :
```